core/src/main/kotlin/br/com/concrete/yosef/api/property/size/HeightCommand.kt

```diff
@@ -22,19 +22,25 @@ class HeightCommand : DynamicPropertyCommand {
     }
 
     override fun apply(view: View, dynamicProperty: DynamicProperty) {
-        if (dynamicProperty.type == "dimenSpec") {
-            when (dynamicProperty.value) {
-                MATCH -> view.layoutParams.height = ViewGroup.LayoutParams.MATCH_PARENT
-                WRAP -> view.layoutParams.height = ViewGroup.LayoutParams.WRAP_CONTENT
-                else -> throw IllegalArgumentException("Can't apply ${dynamicProperty.name}" +
-                    " with value ${dynamicProperty.value}.")
-            }
-        } else if (dynamicProperty.type == "dimen") {
-            view.layoutParams.height = dynamicProperty
+        when (dynamicProperty.type) {
+            "dimenSpec" -> applyDimenSpec(view, dynamicProperty)
```

Comment:
I like this but since we're into refactoring, why not extract both constants from here?